tree/src/mv.rs

```diff
@@ -75,15 +80,20 @@ fn prompt_user(prompt: &str) -> bool {
     response.to_lowercase().starts_with('y')
 }
 
-fn move_file(cfg: &Config, source: &str, target: &str) -> io::Result<()> {
+fn move_file(
+    cfg: &Config,
+    source: &Path,
+    target: &Path,
+    created_files: Option<&HashSet<PathBuf>>,
+) -> io::Result<()> {
     // 1. If the destination path exists, conditionally prompt user
     let target_md = match fs::metadata(target) {
         Ok(md) => Some(md),
         Err(e) => {
             if e.kind() == io::ErrorKind::NotFound {
                 None
             } else {
-                eprintln!("{}: {}", target, e);
+                eprintln!("{}: {}", target.to_string_lossy(), e);
```

Comment:
Prefer to standardize on `.display()` for outputting Path and PathBuf: https://doc.rust-lang.org/std/path/struct.Path.html#method.display